pkg/update/update_cockroach_version.go

```diff
@@ -251,6 +251,7 @@ func preserveDowngradeSetting(ctx context.Context, db *sql.DB) (*semver.Version,
 		return nil, errors.Wrapf(err, "getting preserve downgrade option failed")
 	}
 	if preserveDowngradeSetting == "" {
+		fmt.Println("AICI")
```

Comment:
replace with logging statement

Reply:
sorry ... my playground ... I removed it